tests/RepositoryForms/Limitation/DataTransformer/UDWBasedValueModelTransformerTest.php

```diff
@@ -65,29 +65,30 @@ public function dataProviderForTransform(): array
         ];
     }
 
-    /**
-     * @dataProvider dataProviderForTransformThrowsTransformationFailedException
-     */
-    public function testTransformThrowsTransformationFailedException(string $exceptionClass)
+    public function testTransformThrowsTransformationFailedException(): void
     {
         $this->expectException(TransformationFailedException::class);
 
         $this->locationService
             ->expects($this->any())
             ->method('loadLocation')
             ->willThrowException(
-                $this->createMock($exceptionClass)
+                $this->createMock(UnauthorizedException::class)
             );
 
         $this->transformer->transform(['/1/2/54']);
     }
 
-    public function dataProviderForTransformThrowsTransformationFailedException(): array
+    public function testTransformWithDeletedLocation(): void
     {
-        return [
-            [NotFoundException::class],
-            [UnauthorizedException::class],
-        ];
+        $this->locationService
+            ->expects($this->any())
```

Comment:
AFAIK `self::any()` means nothing in this context and can be omitted.

Otherwise use `self::atLeastOnce()` for example to add assertion about the location service being actually used.

Reply:
```suggestion
            ->expects(self::atLeastOnce())
```

or

```suggestion
```